### PR TITLE
Text-CSV with foss2016a

### PR DESCRIPTION
**Pull request by RvDijk**

### CODE REVIEW

**Comment by RvDijk** [comment]
@boegel Is there any way to get perlmajver in the sanitychecks instead of using "perl5"?

**Comment by boegel** [comment]
@RvDijk you can, but I don't think it's worth it for Perl, since Perl 6.x is very different from Perl 5.x, so 'hardcoding' `perl5` is fine by me

File: easybuild/easyconfigs/t/Text-CSV/Text-CSV-1.33-foss-2016a.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+easyblock = 'PerlModule'
+
+name = 'Text-CSV'
+version = '1.33'
+#versionsuffix = '-Perl-%(perlver)s-bare'
```
Comment (boegel) [inline easybuild/easyconfigs/t/Text-CSV/Text-CSV-1.33-foss-2016a.eb]: @RvDijk why comment this out, it's still relevant even if the Perl dep is a bare variant?

File: easybuild/easyconfigs/t/Text-CSV/Text-CSV-1.33-foss-2016a-Perl-5.22.1.eb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+#versionsuffix = '-Perl-%(perlver)s-bare' 
+
+homepage = 'https://metacpan.org/pod/Text::CSV'
+description = """Text-CSV parser"""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+source_urls = ['https://cpan.metacpan.org/authors/id/M/MA/MAKAMAKA/']
+sources = [SOURCE_TAR_GZ]
+
+
```
Comment (boegel) [inline easybuild/easyconfigs/t/Text-CSV/Text-CSV-1.33-foss-2016a-Perl-5.22.1.eb]: @RvDijk please remove duplicate empty line...

File: easybuild/easyconfigs/t/Text-CSV/Text-CSV-1.33-foss-2016a-Perl-5.22.1.eb [hunk 3]
```diff
@@ -0,0 +1,26 @@
+easyblock = 'PerlModule'
+
+name = 'Text-CSV'
+version = '1.33'
+#versionsuffix = '-Perl-%(perlver)s-bare'
```
Comment (boegel) [inline easybuild/easyconfigs/t/Text-CSV/Text-CSV-1.33-foss-2016a-Perl-5.22.1.eb]: @RvDijk still commented out?

File: easybuild/easyconfigs/t/Text-CSV/Text-CSV-1.33-foss-2016a-Perl-5.22.1.eb [hunk 4]
```diff
@@ -0,0 +1,26 @@
+easyblock = 'PerlModule'
+
+name = 'Text-CSV'
+version = '1.33'
+#versionsuffix = '-Perl-%(perlver)s-bare' 
+
+homepage = 'https://metacpan.org/pod/Text::CSV'
+description = """Text-CSV parser"""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
```
Comment (boegel) [inline easybuild/easyconfigs/t/Text-CSV/Text-CSV-1.33-foss-2016a-Perl-5.22.1.eb]: empty line below please

File: easybuild/easyconfigs/t/Text-CSV/Text-CSV-1.33-foss-2016a-Perl-5.22.1.eb [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+source_urls = ['https://cpan.metacpan.org/authors/id/M/MA/MAKAMAKA/']
+sources = [SOURCE_TAR_GZ]
+
+
+dependencies = [
+    ('Perl', '5.22.1', '-bare')
```
Comment (boegel) [inline easybuild/easyconfigs/t/Text-CSV/Text-CSV-1.33-foss-2016a-Perl-5.22.1.eb]: @RvDijk are you sure you want to go with `bare` here?

Comment (RvDijk) [inline easybuild/easyconfigs/t/Text-CSV/Text-CSV-1.33-foss-2016a-Perl-5.22.1.eb]: @boegel I'll use the normal Perl version, bare fails indeed.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2006.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/fee8bbae89a4abd026872a1f4a45b344 for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Going in, thanks @RvDijk!

**Pull request merged**